src/html.js

```diff
@@ -51,6 +51,17 @@ export default class HTML extends React.Component {
             }
             rel='stylesheet'
           />
+          <link
+            href={
```

Comment:
So confirm this isn't an always thing, you only use that method when it's longer than 80 characters. If that's the case I'll make the needed adjustment.